### ISSUE TITLE
sqlx is  coroutine safe?

### ISSUE THREAD

**Issue by toplinuxsir** [jmoiron/sqlx#516]
sqlx is  coroutine safe?

**Comment by didil** [comment]
yes. I'd advise to read the guite at http://jmoiron.github.io/sqlx/ it's really good
> ... it can be safely used for concurrent querying ...
